contracts/MerkleLib.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+
+
+/**
+ * @notice Library to help with merkle roots, proofs, and claims.
+ */
+library MerkleLib {
+    // TODO: some of these data structures can be moved out if convenient.
+    // This data structure is used in the settlement process on L1.
+    // Each PoolRepayment structure is responsible for balancing a single chain's SpokePool across all tokens.
+    struct PoolRepayment {
```

Comment:
nit: repayment sounds like paying back the relayer. Why not use another more clear word like "rebalance"?

Reply:
Good point. Will update.

Reply:
Fixed.